### PR TITLE
Modify the regex to extract stats in info popup to support i18n

### PR DESCRIPTION
**Pull request by lalitpatel**
Required for #48

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -448,8 +448,16 @@
...
-            text = text.replace(/(.+?\(‹cost›\))(.+?)<i>\s*(Upgrades:.+?)<\/i>(.*)/m,
+        } else if (type === "UNIT" || type === "UNIQUEUNIT" ) {
+            text = text.replace(/(.+?\(‹cost›\))(.+?)<i>\s*(.+?)<\/i>(.*)/m,
+                '<p>$1</p>' +
+                '<p>$2</p>' +
+                '<p><em>$3</em></p>' +
+                '<p class="helptext__stats">$4</p>');
+        } else if (type === "BUILDING") {
+            // convert the "Required for" text in <i> to <em> so that it doesn't break the next regex
+            text = text.replace(/<b><i>(.+?)<\/b><\/i>/m, '<b><em>$1</em></b>');
```
Comment (HSZemi) [inline index.html:459]: smart

**Comment by HSZemi** [comment]
While we are messing with this regex, would it be possible to fix #52 here as well?

**Comment by lalitpatel** [comment]
Sure. Let me address that.

**Comment by HSZemi** [comment]
I just clicked through all panels (except all unique units / techs): Spies/Treason and the three Age upgrades do not look properly formatted as well. Maaaaaybe we can also fix that :no_mouth:

**Comment by lalitpatel** [comment]
Thanks for the through QA 😄 

The Age upgrades are easy to fix. For Spies/Treason there is no place holder fo `‹cost›` so I will have to handle it separately. Let me try to take a stab.

**Comment by lalitpatel** [comment]
Fixed for Age upgrades and Spies/Treason. For Wonders, pls see my comment [here](https://github.com/SiegeEngineers/aoe2techtree/issues/52#issuecomment-628532085).

> @HSZemi I was going through [this page](https://ageofempires.fandom.com/wiki/Wonder_(Age_of_Empires_II)#Further_statistics). It seems Wonders do get affected by upgrades like Town Watch, Masonry etc. Dose it mean that the data in the `data.json` file is incorrect for Wonders?

File: index.html [hunk 2]
```diff
@@ -462,6 +462,13 @@
                 '<p>$2</p>' +
                 '<p><em>$3</em></p>' +
                 '<p class="helptext__stats">$4</p>');
+            if (parseInt(id) === 276) {
+                // Handle Wonders separately as the upgrades text is missing for them.
```
Comment (HSZemi) [inline index.html:466]: Do we want to be brave here and not check for a specific ID, but for the absence of `<i>`?

Comment (lalitpatel) [reply index.html]: Makes sense. That way if the info is fixed for Wonders, it would not require any changes.

**Comment by HSZemi** [comment]
Thank you :yellow_heart:

**Comment by pardoman** [comment]
Thank you!

**Pull request merged**